api/doc/openapi.yaml

```diff
@@ -385,6 +385,74 @@ paths:
           $ref: '#/components/responses/404'
         "500":
           $ref: '#/components/responses/500'
+  /state/module/{address}/{module_name}:
```

Comment:
I think ‘Account/{address}/module/{module}’ would be the restful conjugation of the existing plural ‘Account/{address}/modules’

Reply:
good point.. moving them over